Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.14.2-alpine
+FROM node:18.14.2-slim
```

Comment:
alpine build fails in hard to debug ways when run in CI or when targeting amd64 while on arm64